### PR TITLE
Adding standardized styling files 

### LINKED ISSUE #26: Add standardized styling
- Create constants for colours, typography, etc in a separate styles folder

### PR DESCRIPTION
**Pull request by lesliexin**
**Summary**
Closes #26. Adding in standard styles files to organize styling. Controls the base styling across the entire application with just a few files which makes refactoring incredibly simple. In addition, the design system is easily reflected in the source code. 

**What has changed:**
- Added `styles` folder
- Added in files for typography, device sizes (responsiveness), and colours 
- Edited the existing components to adopt the new styling

### CODE REVIEW

File: ui/src/styles/typography.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  text-align: center;
+  color: ${colours.accentColour};
+
+  @media ${device.mobileS} {
+    font-size: ${smallText};
+  }
+`;
+
+export const typography = {
+    TitleText: TitleText,
```
Comment (AbhiPrasad) [inline ui/src/styles/typography.js]: Hmm why did we decide to export an object this way?

I'd rather just export each component separately and then import the component from `styles/typography`. This works better with tree shaking btw.

File: ui/src/styles/colours.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const white = "#FFFFFF";
+const black = "#000000";
+const pink = '#E9A9A9';
+
+const colours = {
```
Comment (AbhiPrasad) [inline ui/src/styles/colours.js]: +1 on this, good stuff.

File: ui/src/pages/ShoeMap.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
+import {typography} from "../styles";
 
 export const ShoeMap = ()=> {
     return (
         <div>
-          <h1>ShoeMap</h1> 
-          <span> test text fkjbkjbjdbvdjfv</span> 
+          <typography.TitleText>ShoeMap</typography.TitleText>
```
Comment (AbhiPrasad) [inline ui/src/pages/ShoeMap.tsx]: I'd rather the pattern be import this from a file itself, it helps tree shaking + readability.

Destructuring >> object access IMO.

File: ui/src/pages/Home.tsx [hunk 4]
```diff
@@ -1,26 +1,21 @@
...
-  font-size: 3em;
-  text-align: center;
-  color: palevioletred;
-`;
+import {colour, typography} from "../styles";
 
 const Button = styled.button`
   display: flex;    
-  color: palevioletred;
+  color: ${colour.accentColour};
```
Comment (AbhiPrasad) [inline ui/src/pages/Home.tsx]: color vs colour lol

I'm of the mind to 🇺🇸  everything, but I'm fine with this way too.

Comment (lesliexin) [reply ui/src/pages/Home.tsx]: loool I thought long and hard about this and went with 🇨🇦 but i'm fine w/ either

File: ui/src/styles/typography.js (hunk 1 above)
Comment (lesliexin) [reply ui/src/styles/typography.js]: Only reason why is we can easily change the typography of something within the file. What's currently implemented is a bad example but if we had: 
`export const typography = {
    TitleText: LargeBlackOpenSans,
}`
 and wanted to change the style of the title, we could just: 
`export const typography = {
    TitleText: LargeBlueInter,
}`
versus having to change all occurrences throughout the code. Having the exports this way enables us to componentize. 

Buttttt at the same time I do agree that it has its cons.

Comment (AbhiPrasad) [reply ui/src/styles/typography.js]: Ah gotcha, I can see the reasoning then.

I'm of the mind that this project won't get big enough so we probably don't need to prematurely optimize this stuff, even if that means having to find-and-replace like 5-6 component names, but your call, feel free to keep as is.

**Review by AbhiPrasad — APPROVED** [review]
lets ship it

File: ui/src/styles/typography.js (hunk 1 above)
Comment (lesliexin) [reply ui/src/styles/typography.js]: Hmm I agree, I'll change it!

File: ui/src/pages/Home.tsx (hunk 4 above)
Comment (lesliexin) [reply ui/src/pages/Home.tsx]: changed my mind 😅

**Pull request merged**